docs/UserGuide.md

```diff
@@ -489,6 +602,8 @@ Format: `exit`
 Wishful Shrinking's data is saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
```

Comment:
Might want to consider changing this to Header 1 instead of Header 2 cause if Header 2 in toc this might appear under Feature section.